src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
@aliatayee as I mentioned earlier in your books reducer kindly ensures your definition follows the ducks' pattern.